ddl/ddl_worker.go

```diff
@@ -85,6 +85,9 @@ func cleanNotify(ch chan struct{}) {
 func (d *ddl) isOwner() bool {
 	isOwner := d.ownerManager.IsOwner()
 	log.Debugf("[ddl] it's the job owner %v, self id %s", isOwner, d.uuid)
+	if isOwner {
+		metrics.DDLCounter.WithLabelValues(metrics.IsDDLOwner).Inc()
```

Comment:
what does this counter for?
It seems that this function will be called for every ddl sql,

Reply:
We could find which tidb-server is owner from grafana. It is easier that use the `admin show ddl` statement.